### PR TITLE
Update use-dkim-to-validate-outbound-email.md

### PR DESCRIPTION
**Pull request by msbemba**
Added another line about configuring DKIM for the first time

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3076

File: microsoft-365/security/office-365-security/use-dkim-to-validate-outbound-email.md [hunk 1]
```diff
@@ -205,6 +205,8 @@ Once you have published the CNAME records in DNS, you are ready to enable DKIM s
 
 5. Select the domain for which you want to enable DKIM and then, for **Sign messages for this domain with DKIM signatures**, choose **Enable**. Repeat this step for each custom domain.
 
+6.  If you are configuring DKIM for the first time and see the error 'No DKIM keys saved for this domain.' you will have to use Windows powershell to enable Dkim Signing as explained in the next step.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/use-dkim-to-validate-outbound-email.md]:
```suggestion
6. If you are configuring DKIM for the first time and see the error 'No DKIM keys saved for this domain.' you will have to use Windows PowerShell to enable DKIM Signing as explained in the next step.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

File: microsoft-365/security/office-365-security/use-dkim-to-validate-outbound-email.md [hunk 2]
```diff
@@ -205,6 +205,8 @@ Once you have published the CNAME records in DNS, you are ready to enable DKIM s
 
 5. Select the domain for which you want to enable DKIM and then, for **Sign messages for this domain with DKIM signatures**, choose **Enable**. Repeat this step for each custom domain.
 
+6. If you are configuring DKIM for the first time and see the error 'No DKIM keys saved for this domain.' you will have to use Windows PowerShell to enable DKIM Signing as explained in the next step.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/use-dkim-to-validate-outbound-email.md]:
```suggestion
6. If you are configuring DKIM for the first time and see the error 'No DKIM keys saved for this domain.' you will have to use Windows PowerShell to enable DKIM signing as explained in the next step.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Apologies @msbemba, one more minor edit, please.

**Review by Josegto1 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MSFTTracyP Please merge this PR. Thanks.

**Review by MSFTTracyP — COMMENTED** [review]
"If you are configuring DKIM for the first time and see the error 'No DKIM keys saved for this domain.' you"

So if you're quoting an error message, don't include the period mark. It can be confusing.

**Pull request merged**